### ISSUE TITLE
The predicted location seems to be very different from the groundtruth...

### ISSUE THREAD

**Issue by hnyz979** [Guanghan/ROLO#30]
Hi, Dr. Ning
I am now trying your first step, namely, simply reproducing the results with the pre-trained model with the "ROLO_network_test_all.py" code. I have downloaded "model_demo.ckpt" and ran the codes. The codes seems to be working, but I am a little confused about the results:
(The following are the results of the first and second iteration, and I uncommented the codes to print out "batch_ys" and "pred_location", which seems to be the ground truth and the predicted location. However, the results are as follows:)
('Batch_ys: ', array([[ 0.51145833,  0.64296875,  0.19791667,  0.5078125 ]]))
('ROLO Pred: ', array([[-0.07923603, -0.10261807,  0.09152682,  0.03067834]], dtype=float32))
('len(pred) = ', 1)
('ROLO Pred in pixel: ', -38.033294677734375, -65.675563812255859, 43.932874202728271, 19.634140729904175)
('correct_prediction int: ', array([[0, 0, 0, 0]]))
Iter 0, Minibatch Loss= 28.594887

('Batch_ys: ', array([[ 0.51354167,  0.64296875,  0.19791667,  0.5078125 ]]))
('ROLO Pred: ', array([[-0.07972586, -0.10288554,  0.09197099,  0.03046498]], dtype=float32))
('len(pred) = ', 1)
('ROLO Pred in pixel: ', -38.268413543701172, -65.846743583679199, 44.146074056625366, 19.497586488723755)
('correct_prediction int: ', array([[0, 0, 0, 0]]))
Iter 1, Minibatch Loss= 28.683752

It seems that, the Batch_ys(groundtruth) are very different from the ROLO Pred(ROLO predictions). Could you kindly tell me where did I do wrong? Thank you very much! @Guanghan

**Comment by lihaixiang** [comment]
@hnyz979 我也遇到和你同样的问题
实际上，在经过一段时间的尝试后，我无法重复论文中所描述的预测结果，无论是实验1还是实验2。我已经转向其他work

**Comment by hnyz979** [comment]
@lihaixiang 你试过YOLO没？那个能跑起来吗？

**Comment by lihaixiang** [comment]
可以跑起来

**Comment by lihaixiang** [comment]
@hnyz979 可以

**Comment by hnyz979** [comment]
@lihaixiang  cool~~我准备下一个https://github.com/DrewNF/Tensorflow_Object_Tracking_Video试试。刚刚进入追踪这个领域，感觉找到个能跑起来的不容易啊。

**Comment by wanjinchang** [comment]
@hnyz979，你好！您提到的那个工程，看了下好像目前还没加上跟踪吧？我也是最近才接触到目标跟踪这个领域，希望能得到你的回复！
